assets/questions/pirateTranslator.js

```diff
@@ -224,6 +273,61 @@ globalData.questions['pirateTranslator'] = {  // eslint-disable-line dot-notatio
           "Don't forget spaces in your output."
         ].join('')
       ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.alwaysPrependSpace',
+      ignoredTestSuiteIds: [],
+      messages: [
+        [
+          'Spaces should only appear between words, not at the ',
+          'beginning of the sentence.'
+        ].join('')
+      ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.alwaysAppendSpace',
+      ignoredTestSuiteIds: [],
+      messages: [
+        [
+          'Spaces should only appear between words, not at the ',
+          'end of the sentence.'
+        ].join('')
+      ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.onlySpaceAfterPirateWords',
+      ignoredTestSuiteIds: [],
+      messages: [
+        [
+          'Spaces should only appear between each whether English ',
+          'or Pirate.'
+        ].join('')
+      ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.outputingStringIndexes',
+      ignoredTestSuiteIds: [],
+      messages: [
+        [
+          'You need to output words, not word indexes. When you ',
```

Comment:
Maybe just give the first sentence as the first feedback? Then if they get it wrong again, repeat the first sentence and give the second sentence as well?

(Note that, in general, if "messages" has more than one element, then the first element is shown the first time the error is detected. If the error is detected a second time after the student tries to fix their code, then the second element is shown. And so on.)